<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SteamTargetUserWindowClass</class>
 <widget class="QMainWindow" name="SteamTargetUserWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>137</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>282</width>
    <height>137</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>282</width>
    <height>137</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GloSC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="checkBox_showOverlay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Overlay</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_enableControllers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Virtual Controllers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resetControllers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Controllers / Redetect real controllers</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_showDebugConsole">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show debug console</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SteamTargetUserWindow.qrc"/>
 </resources>
 <connections/>
</ui>
